Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "FMI"
 uuid = "14a09403-18e3-468f-ad8a-74f8dda2d9ac"
 authors = ["TT <tobias.thummerer@informatik.uni-augsburg.de>", "LM <lars.mikelsons@informatik.uni-augsburg.de>", "JK <josef.kircher@student.uni-augsburg.de>"]
-version = "0.3.9"
+version = "0.3.10"
```

Comment:
@ThummeTo FYI This was supposed to be breaking release due to the changes in the causality, variability and initial enums.